bench/wtperf/wtperf.h

```diff
@@ -66,14 +66,15 @@ typedef struct {
     uint64_t throttle; /* Maximum operations/second */
                        /* Number of operations per transaction. Zero for autocommit */
     int64_t ops_per_txn;
-    int64_t pause;            /* Time between scans */
-    int64_t read_range;       /* Range of reads */
-    int32_t table_index;      /* Table to focus ops on */
-    int64_t truncate;         /* Truncate ratio */
-    uint64_t truncate_pct;    /* Truncate Percent */
-    uint64_t truncate_count;  /* Truncate Count */
-    int64_t modify_delta;     /* Value size change on modify */
-    int64_t update_delta;     /* Value size change on update */
+    int64_t pause;           /* Time between scans */
+    int64_t read_range;      /* Range of reads */
+    int32_t table_index;     /* Table to focus ops on */
+    int64_t truncate;        /* Truncate ratio */
+    uint64_t truncate_pct;   /* Truncate Percent */
+    uint64_t truncate_count; /* Truncate Count */
+    int64_t modify_delta;    /* Value size change on modify */
```

Comment:
This should sort with the other `modify_` fields. Preferably alphabetized up above, but at a minimum together.

Reply:
Changed. I moved all the newly added bool also together. Is there any coding guidelines that we should follow the structure members order based on size?